### DISCUSSION TITLE
mac - No such file / Architecture issue on #include <sodium.h>

### DISCUSSION THREAD

**Discussion by rkytopen** [jedisct1/libsodium#1169] [Q&A]
Hello- 
I am currently upgrading from an Intel-based Catalina system to an M1-based Monterey system, and I'm now receiving the following when trying to run a previously-working script:

‘/usr/local/opt/libsodium/lib/libsodium.23.dylib’ (no such file),
‘/usr/local/lib/libsodium.23.dylib’ (mach-o file, but is an incompatible architecture (have ‘arm64’, need ‘x86_64’)),
‘/usr/lib/libsodium.23.dylib’ (no such file)

Any advice? My first install passed the make check with no errors, reinstalling generated the attached error log.
[test-suite.log](https://github.com/jedisct1/libsodium/files/8467946/test-suite.log)

**Comment by jedisct1** [comment]
On M1 systems, Homebrew installs software in `/opt/homebrew` not `/usr/local` any more.
